app/src/main/res/layout/pet_favorites_items.xml

```diff
@@ -34,6 +34,16 @@
         android:textIsSelectable="false" />
 
     <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentRight="true"
+        android:id="@+id/tvRemovePetFav"
+        android:layout_marginTop="5dp"
+        android:textColor="#363636"
+        android:drawableLeft="@drawable/ic_remove_petfav"
+        android:gravity="center_vertical"/>
+
```

Comment:
maybe you did this in one of the other pull requests but I think we'd want the unlike button to be an actual button and not  background image on a text view. Checkout the ImageButton element which you can specify in an xml file. 

That will make it easier to have press states on the button as well.